### PR TITLE
Updated liquidate event

### PR DESCRIPTION
**Pull request by this-username-is-taken**

### CODE REVIEW

**Review by 0xbe1 — COMMENTED** [review]
lgtm

File: schema-lending.graphql [hunk 1]
```diff
@@ -994,6 +994,9 @@ type Liquidate implements Event @entity {
   " Address that carried out the liquidation "
   from: String!
 
+  " Address that got liquidated "
+  liquidated: String!
```
Comment (0xbe1) [inline schema-lending.graphql:998]: does `liquidatee` sound less ambiguous? `liquidated` could also mean the liquidated asset aka collateral

Comment (this-username-is-taken) [reply schema-lending.graphql:998]: I don't think `liquidatee` is a real word :)

Comment (this-username-is-taken) [reply schema-lending.graphql:998]: Also, `liquidated` is a bit ambiguous

Comment (0xbe1) [reply schema-lending.graphql:998]: or just `liquidatedAccount` or even `victim` 😆

**Pull request merged**